docs/rules/Community-rules.md

```diff
@@ -0,0 +1,24 @@
+---
+id: community-rules
+sidebar_label: Community rules
+title: Community rules
+---
+
+Many rules have been developed by the community.
+Follow [this documentation](external-rules.md) to use them in your project.
+
+
+## Available rules:
+*[Edit](https://github.com/scalacenter/scalafix/edit/master/docs/rules/community-rules.md) 
```

Comment:
looks like the file name is still upper-case, does it work?

Reply:
ah thanks